package.json

```diff
@@ -20,11 +20,13 @@
     "eslint-plugin-jest": "^22.4.1",
     "eslint-plugin-node": "^8.0.1",
     "expect": "^29.2.1",
-    "mocha": "^10.1.0",
+    "mocha": "^10.3.0",
     "nodemon": "^2.0.20",
     "supertest": "^6.3.0"
   },
   "dependencies": {
-    "express": "^4.18.2"
+    "cors": "^2.8.5",
+    "express": "^4.18.3",
+    "uuid": "^9.0.1"
```

Comment:
I see that you installed `uuid`, then maybe better to use it or uninstall if you changed your mind